### DISCUSSION TITLE
Is it possible to finetune from the CLI a model that requires changing its last layer?

### DISCUSSION THREAD

**Discussion by adosar** [speediedan/finetuning-scheduler#14] [General]
First of all, congrats for this very interesting package!

I was reading the docs for how to fine-tune the model from the CLI and I was wondering if someone can fine-tune in the following case.

A model was trained to predict 5 classes but now it must be fine-tuned to predict 10 classes. This means that the final layer must be replaced before we start the fine-tuning. Is it possible to achieve that from the CLI?
